vignettes/articles/InputFiles.Rmd

```diff
@@ -63,21 +75,15 @@ lucEmissions=csv:input/emissions/ssp245_emissions.csv
 
 In this case, the `input/emissions/ssp245_emissions.csv` file is parsed and the data in column `lucEmissions` are assigned to the model variable, with a required `Date` column providing the time index.
 
-**Table of variable inputs (with a time index)**
```

Comment:
Add a comment about how the "(csv)" entry in the Default column indicates that the values are passed in via a path to a csv file